lib/apis/graphql/v2/observeDomainEvents/getTypeDefinitions.ts

```diff
@@ -0,0 +1,23 @@
+import { getDomainEventSchema } from '../../shared/schemas/getDomainEventSchema';
+import { getGraphqlFromJsonSchema } from 'get-graphql-from-jsonschema';
+import { Schema } from '../../../../common/elements/Schema';
+import { stripIndent } from 'common-tags';
+
+const getTypeDefinitions = function (): string {
+  const domainEventSchema: Schema = getDomainEventSchema();
+  const domainEventGraphQL = getGraphqlFromJsonSchema({
```

Comment:
To be consistent with everywhere else, this should be:

```suggestion
  const domainEventGraphQl = getGraphqlFromJsonSchema({
```